internal/gatewayapi/route.go

```diff
@@ -271,6 +271,11 @@ func (t *Translator) processHTTPRouteRules(httpRoute *HTTPRouteContext, parentRe
 	return routeRoutes, nil
 }
 
+func processHTTPRouteRule(irRoute *ir.HTTPRoute, rule gwapiv1.HTTPRouteRule) {
```

Comment:
suggest using a different name here, since we have `processHTTPRouteRules` https://github.com/envoyproxy/gateway/blob/00ecd08a789e3b443bc15561a10461f4d6816630/internal/gatewayapi/route.go#L183 which processes everything, something like `processRouteTrafficFeatures` maybe ?